build-faiss-linux.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-FAISS_VERSION=${1:-v1.7.1}
+FAISS_VERSION=${1:-main}
```

Comment:
Is there no release tag for the version you are wanting to use right now?

Reply:
unfortunately there's no tag for this version. i can try against their latest release tag and see if it has any impact on the timings.